explainers/subresource-loading.md

```diff
@@ -194,6 +195,35 @@ will have its
 set to "`webbundle`"
 ([whatwg/fetch#1120](https://github.com/whatwg/fetch/issues/1120)).
 
+## Content Security Policy (CSP)
+
+For resources loaded from bundles, URL matching of CSP is done based on the URL
+of the resource, not the URL of the bundle. For example, given this CSP header:
+```
+Content-Security-Policy: script-src https://example.com/script/; frame-src urn:
+```
+
+In the following, the first `<script>` and the `<iframe>` will be loaded, and
+the second `<script>` will be blocked:
+
+```
+<link rel="webbundle"
+  href="https://example.com/subresources.wbn"
+  resources="https://example.com/script/a.js
+             https://example.com/b.js
+             urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"
+/>
+
+<script src=”https://example.com/script/a.js”></script>
+<script src=”https://example.com/b.js”></script>
+<iframe src="urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6"></iframe>
```

Comment:
Give an example of how a `urn:uuid:` script is blocked?

Reply:
Updated the example to show that `*` does not allow a `urn:uuid:` resource.